cluster/up.sh

```diff
@@ -25,15 +25,23 @@ if [[ "$KUBEVIRT_PROVIDER" != external ]]; then
     $(kubevirtci::path)/cluster-up/up.sh
 fi
 
-# Deploy CNA
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/namespace.yaml
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/network-addons-config.crd.yaml
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/operator.yaml
-./cluster/kubectl.sh create -f ./hack/cna/cna-cr.yaml
-
-# wait for cluster operator
-./cluster/kubectl.sh wait networkaddonsconfig cluster --for condition=Available --timeout=800s
-
+if [[ "$CNAO_PROVIDER" != external ]]; then
```

Comment:
If you have else, you should do the positive check first.

Where do you set the default value?

Reply:
Let's use `CNAO_VERSION` instead of `CNAO_PROVIDER` here by default it has the CNAO_VERSION we use at project but can contain `external` value meaning we don't install it.

Reply:
ack